core/src/main/scala/kafka/server/Server.scala

```diff
@@ -46,6 +46,22 @@ object Server {
     new Metrics(metricConfig, reporters, time, true, metricsContext)
   }
 
+  def initializeMetrics(
+    config: KafkaConfig,
+    time: Time,
+    metaProps: MetaProperties
```

Comment:
It seems to me the properties in ```MetaProperties``` is duplicate to ```KafkaConfig``` in this case. Is there any reason that we need to pass ```MetaProperties```?

Reply:
@chia7712 We need the clusterId from `MetaProperties`. I modified this to pass it through directly avoid the duplicate methods.